Makefile

```diff
@@ -76,44 +82,61 @@ react-start:
 react-lint:
 	npm run lint --workspace packages/cpt-ui
 
-cdk-deploy: guard-stack_name
+cdk-deploy: guard-service_name guard-CDK_APP_NAME
 	REQUIRE_APPROVAL="$${REQUIRE_APPROVAL:-any-change}" && \
 	VERSION_NUMBER="$${VERSION_NUMBER:-undefined}" && \
 	COMMIT_ID="$${COMMIT_ID:-undefined}" && \
 		npx cdk deploy \
-		--app "npx ts-node --prefer-ts-exts packages/cdk/bin/ClinicalPrescriptionTrackerApp.ts" \
+		--app "npx ts-node --prefer-ts-exts packages/cdk/bin/$$CDK_APP_NAME.ts" \
 		--all \
 		--ci true \
 		--require-approval $${REQUIRE_APPROVAL} \
-		--context stackName=$$stack_name \
+		--context serviceName=$$service_name \
 		--context VERSION_NUMBER=$$VERSION_NUMBER \
 		--context COMMIT_ID=$$COMMIT_ID 
 
-cdk-synth:
+cdk-synth: cdk-synth-stateful-resources cdk-synth-stateless-resources
+
+cdk-synth-stateful-resources:
+	npx cdk synth \
```

Comment:
can we add --quiet param to this to prevent clogging up the output with the template when it is run (same for one below)